### PR TITLE
Add translation and max char limit

### PR DESCRIPTION
**Pull request by cuxs**
Would leave some comments on the code

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -36,25 +39,59 @@ def transcribe_time_stamps(segments: list):
     return string
 
 
-def make_srt_subtitles(segments: list):
+def split_text_by_punctuation(text: str, max_length: int):
```
Comment (cuxs) [inline src/main.py:42]: Well... after working a little bit with this, I found that some phrases were way too long for a subtitle, so I added a max_char value that you can tweak to make the subtitles look better. The way it works is that if the limit is exceeded, it'll look for a punctuation mark, question mark, or exclamation point to break the sentence in two, and split the time proportionally.

File: src/main.py [hunk 2]
```diff
@@ -69,26 +106,27 @@ def index(request: Request):
 async def download_subtitle(
         request: Request,
         file: bytes = File(),
-        model_type: str = "tiny",
+        model_type: str = Form("tiny"),
```
Comment (cuxs) [inline src/main.py:109]: Parameters weren't working.

File: src/templates/index.html [hunk 3]
```diff
@@ -20,6 +19,7 @@ <h1 class="mb-4">Audio to Text Converter</h1>
         <div class="mb-3">
             <label for="model_type" class="form-label">Choose a Whisper model:</label>
             <select id="model_type" name="model_type" class="form-select">
+                <option value="turbo">turbo</option>
```
Comment (cuxs) [inline src/templates/index.html:22]: Added the turbo option as the default

**Comment by Kabanosk** [comment]
Wow! Thank you so much for your contribution! 🙌
I really appreciate the effort you put into this PR - it works perfectly and is a fantastic addition to the project!

**Pull request merged**